src/nginx.go

```diff
@@ -1,41 +1,92 @@
 package main
 
 import (
+	"fmt"
 	sdk_args "github.com/newrelic/infra-integrations-sdk/args"
+	"github.com/newrelic/infra-integrations-sdk/data/metric"
+	"github.com/newrelic/infra-integrations-sdk/integration"
 	"github.com/newrelic/infra-integrations-sdk/log"
-	"github.com/newrelic/infra-integrations-sdk/sdk"
+	"net/url"
 )
 
 type argumentList struct {
 	sdk_args.DefaultArgumentList
-	StatusURL  string `default:"http://127.0.0.1/status" help:"NGINX status URL."`
-	ConfigPath string `default:"/etc/nginx/nginx.conf" help:"NGINX configuration file."`
+	StatusURL        string `default:"http://127.0.0.1/status" help:"NGINX status URL."`
+	ConfigPath       string `default:"/etc/nginx/nginx.conf" help:"NGINX configuration file."`
+	RemoteMonitoring bool   `default:"false" help:"Identifies the monitored entity as 'remote'. In doubt: set to true."`
 }
 
 const (
 	integrationName    = "com.newrelic.nginx"
-	integrationVersion = "1.0.2"
+	integrationVersion = "1.1.0"
+	entityRemoteType   = "nginx"
```

Comment:
According to the MMF document this should be `server`

> Nginx: server - concatenation of hostname and port

Reply:
I'm not sure about this. What you are talking about is `entityName` and here it is defined the `entityType`, isn't it?